scriptmodules/emulators/mupen64plus/0002-GLideN64-force-egl.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/GLideN64/src/Graphics/OpenGLContext/opengl_GLInfo.cpp b/GLideN64/src/Graphics/OpenGLContext/opengl_GLInfo.cpp
+index 9f2de97..745133f 100644
+--- a/GLideN64/src/Graphics/OpenGLContext/opengl_GLInfo.cpp
++++ b/GLideN64/src/Graphics/OpenGLContext/opengl_GLInfo.cpp
+@@ -145,7 +145,7 @@ void GLInfo::init() {
+ 	texture_barrierNV = Utils::isExtensionSupported(*this, "GL_NV_texture_barrier");
+ 
+ 	ext_fetch = Utils::isExtensionSupported(*this, "GL_EXT_shader_framebuffer_fetch") && !isGLES2 && (!isGLESX || ext_draw_buffers_indexed) && !imageTextures;
+-	eglImage = (Utils::isEGLExtensionSupported("EGL_KHR_image_base") || Utils::isEGLExtensionSupported("EGL_KHR_image"));
++	eglImage = true; //(Utils::isEGLExtensionSupported("EGL_KHR_image_base") || Utils::isEGLExtensionSupported("EGL_KHR_image"));
```

Comment:
I'm reasonably happy with the PR, except for this hack. @hhromic has been doing some digging into this, so I'd appreciate if a better solution could be found.

(the hack does seem to work absolutely fine, though, so if nobody objected, we could just merge this as-is and then remove/change the patch later when a more elegant fix is found).